accounts/usbwallet/ledger/tokens.go

```diff
@@ -0,0 +1,102 @@
+package ledger
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/binary"
+	"errors"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const TokensBlob = "AAAAaARjVVNEdl3oFoRYYedaJfyhIrtomLixKCoAAAASAACk7DBFAiEApwQFHNBKXp+V2jq8BMD2y/5AwC9bhPQ2H4hT/vMl/B4CIFalOVtBFGREUKMU/F5vDlJLeQrTn6GQeDertpB2FpMvAAAAZwRjR0xERx7ON1DaI3+TuOM5xTaYm4l4pDgAAAASAACk7DBEAiAtUE03OVEDTuAgMR8CeUiXVB7Uqa+4gQHaZtXsU0tjMQIgTsY5//96mGMOtyoQFgjDrh3kztxlUO90MJOLElo6baQ=" // #nosec
```

Comment:
Maybe you could extract it in another file to keep things cleaner. One thing is how to use the data, and the other is the data itself

Reply:
Good idea, I'll do that.